components/Tabs/Tabs.js

```diff
@@ -2,50 +2,45 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
-    
+    this.element= element;    
     // Get the custom data attribute on the Link
-    // this.data;
+     this.data = this.element.dataset;
     
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
+     this.itemElement = this.data.tab;
```

Comment:
This instruction is actually asking for the element (with `class="tab-item"`) that corresponds to the current `data-tab` value (which you did query for later on in line 12 with `document.querySelector('.tabs-item[data-tab="' + this.itemElement + '"]')`).